### PR TITLE
Document the Istio divert driver

### PR DESCRIPTION
**Pull request by pchico83**
We need to create a doc explaining the divert feature, requirements and use cases.
This PR only adds the technical docs to document the fields in the okteto manifest.

### CODE REVIEW

File: versioned_docs/version-1.5/reference/manifest.mdx [hunk 1]
```diff
@@ -155,37 +155,43 @@ A list of commands to deploy your development environment.
 
 It's usually a combination of `helm`, `kubectl`, and `okteto` commands.
 
-#### Deploy a Docker Compose file:
+#### Deploy a Helm chart
```
Comment (pchico83) [inline versioned_docs/version-1.5/reference/manifest.mdx:158]: Based on the text above, it reads better if we document helm before docker compose

File: versioned_docs/version-1.5/reference/manifest.mdx [hunk 2]
```diff
@@ -214,16 +223,34 @@ deploy:
...
+    - helm upgrade --install movies chart --set api.image=${OKTETO_BUILD_API_IMAGE} --set frontend.image=${OKTETO_BUILD_FRONTEND_IMAGE}
+  divert:
+    driver: istio
+    namespace: staging
+    service: api
+    virtualServices:
+      - frontend
 ```
 
+When divert is enabled, Okteto will add the header `x-okteto-divert` to every request coming from the developer namespace.
```
Comment (rberrelleza) [inline versioned_docs/version-1.5/reference/manifest.mdx:244]: Can you put the 3 paragraphs of explanation before the sample? I think it reads better that way.

**Review by rberrelleza — APPROVED** [review]

**Comment by rberrelleza** [comment]
Does this means that we're ready to fully support divert?

**Comment by pchico83** [comment]
> Does this means that we're ready to fully support divert?

@rberrelleza yes starting from chart 1.6.0. And only for the istio driver. we are still evaluating what to do with the weaver driver

**Pull request merged**